### PR TITLE
integrate with routing in crates.io

### PR DESCRIPTION
**Pull request by ustulation**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @krishnaIndia

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: Cargo.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ homepage      = "http://maidsafe.net"
 
 [dependencies]
 # ---- MaidSafe -----
-#routing         = "0.2.*"
+routing         = "0.3.4"
```
Comment (benjaminbollen) [inline Cargo.toml]: You don't trust routing the upcoming routing 0.3.5 ;-)

Comment (ustulation) [inline Cargo.toml]: doesn't `= "0.3.4"` actually mean `>= "0.3.4"` as opposed to `= "=0.3.4"` which would mean `exactly and only == 0.3.4` ? I'm confused myself .. @Fraser999 do you know ? ^^^

Comment (krishnaIndia) [inline Cargo.toml]: Can't we go with `0.3.*`?

Comment (benjaminbollen) [inline Cargo.toml]: oh no sorry @ustulation as most of the time, you are right; it does mean the minimal version

Comment (Fraser999) [inline Cargo.toml]: The rules vary for how far past `x.y.z` it considers.  For `0.3.4` it will consider anything >= 0.3.4 and < 0.4.0

Comment (ustulation) [inline Cargo.toml]: i think it's the same, atleast for `"0.3"`.. but seems better so will do that in a moment

Comment (Fraser999) [inline Cargo.toml]: Having said that, I think we have a policy of specifying `*` for all non-MaidSafe libs and `x.y.*` for all MaidSafe libs (not sure why).

**Pull request merged**